<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegionalPropertiesWidget</class>
 <widget class="QWidget" name="RegionalPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>40</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbUseInheritable">
        <property name="text">
         <string>Use inheritable settings from parent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbUseOwnRegionalSettings">
        <property name="text">
         <string>Use folder's settings</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Display time using &quot;foreign&quot; time zone settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="chkApplyToGui">
        <property name="text">
         <string>Apply remote site time settings to GUI and reports</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkApplyToSchedules">
        <property name="text">
         <string>Apply remote site time settings to schedules assigned to the test and actions</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,6">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>GMT offset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbGmtOffset"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>173</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
